_layouts/product-overview.html.slim

```diff
@@ -50,6 +53,11 @@ br
     div(class="feature-text")
       = page._features3
 
+- if page._special_table
```

Comment:
Good point.  Originally, it caused problems with the layout.  I will test with detail.